app/questionnaire/router.py

```diff
@@ -369,18 +369,25 @@ def _get_return_to_for_calculated_summary(
             return_to_block_id = ",".join(remaining) if remaining else None
 
             # remove first item and return the remaining ones
-            # Type ignore: return_location.return_to will always be populated at this point
-            return_to_remaining = ",".join(return_location.return_to.split(",")[1:]) or None  # type: ignore
+            # Type ignore: return_location.return_to and return_location.return_to_answer_id will always be populated at this point
+            return_to = ",".join(return_location.return_to.split(",")[1:]) or None  # type: ignore
+            anchor, *return_to_answer_ids = return_location.return_to_answer_id.split(",")  # type: ignore
+            return_to_answer_id = (
+                ",".join(return_to_answer_ids)  # type: ignore
```

Comment:
Do you definitely need this type ignore? I thought mypy might pick it up but I suppose if its coming from a line with a type ignore maybe thats why it doesn't

Reply:
Turns out it wasn't needed,  thanks.